shell/ui/styles.css

```
@@ -728,3 +854,570 @@
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  padding: 14px 16px;
+  color: var(--color-ink-ghost);
+  font: 400 12px/18px var(--font-mono, ui-monospace, monospace);
+}
+[data-iii-ui="shell"] .shui-review-message.warn { color: var(--color-warn); }
+[data-iii-ui="shell"] .shui-review-inline-action {
+  padding: 0;
+  border: 0;
+  border-bottom: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)